### DISCUSSION TITLE
Passler PRTG Desktop

### DISCUSSION THREAD

**Discussion by Stew1973** [Homebrew/discussions#699] [Casks]
Hi,

Not sure if this is the right place but...

Please could I request a cask for Mac

https://www.paessler.com/prtg-desktop

Thank you !

**Comment by carlocab** [comment]
You can contribute one. Here's a guide to help you get started:

https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#how-to-contribute
